graylog2-server/src/main/java/org/graylog/plugins/views/search/SearchJobIdentifier.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.plugins.views.search;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record SearchJobIdentifier(@JsonProperty String id,
```

Comment:
What is the idea behind extracting this into a separate class/record instead of adding just the `executing_node` field to `SearchJob`?

Reply:
Avoiding code duplication.
Besides, it is very likely that future version of `execute` call will return just this identifier.